{"nodes":[{"pos":[459,520],"content":"The exception that is thrown when a generic exception occurs.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is thrown when a generic exception occurs.","pos":[0,61]}]},{"pos":[1656,1751],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Web.HttpUnhandledException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class."},{"pos":[2346,2475],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Web.HttpUnhandledException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified error messages.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class with the specified error messages."},{"pos":[2626,2691],"content":"The message displayed to the client when the exception is thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message displayed to the client when the exception is thrown.","pos":[0,65]}]},{"pos":[3274,3422],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Web.HttpUnhandledException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified error message and inner exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class with the specified error message and inner exception."},{"pos":[3599,3664],"content":"The message displayed to the client when the exception is thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message displayed to the client when the exception is thrown.","pos":[0,65]}]},{"pos":[3738,3824],"content":"The <xref:System.Exception.InnerException*>, if any, that threw the current exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The &lt;xref:System.Exception.InnerException*&gt;, if any, that threw the current exception.","pos":[0,86],"source":"The <xref:System.Exception.InnerException*>, if any, that threw the current exception."}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Web.HttpUnhandledException\n  id: HttpUnhandledException\n  children:\n  - System.Web.HttpUnhandledException.#ctor\n  - System.Web.HttpUnhandledException.#ctor(System.String)\n  - System.Web.HttpUnhandledException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: HttpUnhandledException\n  nameWithType: HttpUnhandledException\n  fullName: System.Web.HttpUnhandledException\n  type: Class\n  summary: The exception that is thrown when a generic exception occurs.\n  syntax:\n    content: 'public sealed class HttpUnhandledException : System.Web.HttpException'\n  inheritance:\n  - System.Exception\n  - System.SystemException\n  - System.Runtime.InteropServices.ExternalException\n  - System.Web.HttpException\n  implements: []\n  inheritedMembers:\n  - System.Runtime.InteropServices.ExternalException.ErrorCode\n  - System.Runtime.InteropServices.ExternalException.ToString\n  - System.Web.HttpException.CreateFromLastError(System.String)\n  - System.Web.HttpException.GetHtmlErrorMessage\n  - System.Web.HttpException.GetHttpCode\n  - System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Web.HttpException.WebEventCode\n  platform:\n  - net462\n- uid: System.Web.HttpUnhandledException.#ctor\n  id: '#ctor'\n  parent: System.Web.HttpUnhandledException\n  langs:\n  - csharp\n  name: HttpUnhandledException()\n  nameWithType: HttpUnhandledException.HttpUnhandledException()\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException()\n  type: Constructor\n  assemblies:\n  - System.Web\n  namespace: System.Web\n  summary: Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class.\n  syntax:\n    content: public HttpUnhandledException ();\n    parameters: []\n  overload: System.Web.HttpUnhandledException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Web.HttpUnhandledException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Web.HttpUnhandledException\n  langs:\n  - csharp\n  name: HttpUnhandledException(String)\n  nameWithType: HttpUnhandledException.HttpUnhandledException(String)\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException(String)\n  type: Constructor\n  assemblies:\n  - System.Web\n  namespace: System.Web\n  summary: Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class with the specified error messages.\n  syntax:\n    content: public HttpUnhandledException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message displayed to the client when the exception is thrown.\n  overload: System.Web.HttpUnhandledException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Web.HttpUnhandledException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Web.HttpUnhandledException\n  langs:\n  - csharp\n  name: HttpUnhandledException(String,Exception)\n  nameWithType: HttpUnhandledException.HttpUnhandledException(String,Exception)\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.Web\n  namespace: System.Web\n  summary: Initializes a new instance of the <xref href=\"System.Web.HttpUnhandledException\"></xref> class with the specified error message and inner exception.\n  syntax:\n    content: public HttpUnhandledException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message displayed to the client when the exception is thrown.\n    - id: innerException\n      type: System.Exception\n      description: The <xref:System.Exception.InnerException*>, if any, that threw the current exception.\n  overload: System.Web.HttpUnhandledException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.Web.HttpException\n  isExternal: false\n  name: System.Web.HttpException\n- uid: System.Web.HttpUnhandledException.#ctor\n  parent: System.Web.HttpUnhandledException\n  isExternal: false\n  name: HttpUnhandledException()\n  nameWithType: HttpUnhandledException.HttpUnhandledException()\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException()\n- uid: System.Web.HttpUnhandledException.#ctor(System.String)\n  parent: System.Web.HttpUnhandledException\n  isExternal: false\n  name: HttpUnhandledException(String)\n  nameWithType: HttpUnhandledException.HttpUnhandledException(String)\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Web.HttpUnhandledException.#ctor(System.String,System.Exception)\n  parent: System.Web.HttpUnhandledException\n  isExternal: false\n  name: HttpUnhandledException(String,Exception)\n  nameWithType: HttpUnhandledException.HttpUnhandledException(String,Exception)\n  fullName: System.Web.HttpUnhandledException.HttpUnhandledException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Web.HttpUnhandledException.#ctor*\n  parent: System.Web.HttpUnhandledException\n  isExternal: false\n  name: HttpUnhandledException\n  nameWithType: HttpUnhandledException.HttpUnhandledException\n"}